packages/shared/src/lib/featureManagement.ts

```diff
@@ -34,9 +34,9 @@ const feature = {
   onboardingChecklist: new Feature('onboarding_checklist', true),
   showCodeSnippets: new Feature('show_code_snippets', false),
   pricingIds: new Feature('pricing_ids', {
-    pri_01jjvm32ygwb1ja7w52e668fr2: PlusPriceType.Yearly,
-    pri_01jbsccbdbcwyhdy8hy3c2etyn: PlusPriceType.Monthly,
-    pri_01jbscda57910yvwjtyapnrrzc: PlusPriceType.Yearly,
+    pri_01jjbwd5j7k0nm45k8e07yfmwr: PlusPriceType.Yearly,
+    pri_01jcdp5ef4yhv00p43hr2knrdg: PlusPriceType.Monthly,
+    pri_01jcdn6enr5ap3ekkddc6fv6tq: PlusPriceType.Yearly,
```

Comment:
Just a note to revert this

Reply:
Yeah this should have no change?

Reply:
Was this changed back?

Reply:
@AmarTrebinjac think you didn't revert this still.

Reply:
@AmarTrebinjac